modules/upgrade-duration-estimate-cluster-upgrade-time.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-upgrade-duration.adoc
+
+:_content-type: REFERENCE
+[id="estimating-cluster-upgrade-time_{context}"]
+= Estimating cluster upgrade time
+
+Historical upgrade duration of similar clusters provides you the best estimate for the future cluster upgrades. However, if the historical data is not available, you can use the following convention to estimate your cluster upgrade time:
+
+----
+Cluster upgrade time = CVO target upgrade payload deployment time + (# node upgrade iterations x MCO node upgrade time)
+----
+
+A node upgrade iteration consists of one or more nodes upgraded in parallel. The control plane nodes are always upgraded in parallel with the worker nodes. In addition, one or more worker nodes can be upgraded in parallel based on the `maxUnavailable` value.
```

Comment:
We need to tell where to find the `maxUnavailable` value. Or what is the value means. because the same name is used in similar context in Kubernetes but it is not exacty same.  I am positive that we have the definition in the current docs. Can we  link it or reuse the definition?